test/bin/scenario.sh

```diff
@@ -620,6 +670,12 @@ action_create() {
     fi
     record_junit "setup" "load_global_settings" "OK"
 
+    if ! load_subscription_manager_plugin; then
+        record_junit "setup" "load_subscription_manager_plugin" "FAILED"
+        return 1
+    fi
+    record_junit "setup" "load_subscription_manager_plugin" "OK"
```

Comment:
Looks like we're unconditionally calling the registration procedure on the VM creation?

Reply:
I think this just sources the functions, but they're called in scenario script

Reply:
Right, this loads the script. The script defines a function. The function is then available to scenarios that need it. The new scenario in this PR uses it.